persistence-demo-test/src_test/com/axonivy/demo/test/PlayerRepoIvyTest.java

```diff
@@ -0,0 +1,59 @@
+package com.axonivy.demo.test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.hibernate.StatelessSession;
+import org.hibernate.internal.SessionFactoryImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.axonivy.demo.PlayerRepo;
+import com.axonivy.demo.PlayerRepo_;
+import com.axonivy.demo.persistence.demo.Player;
+
+import ch.ivyteam.ivy.environment.Ivy;
+import ch.ivyteam.ivy.environment.IvyTest;
+import ch.ivyteam.ivy.scripting.objects.Date;
+
+@IvyTest
+class PlayerRepoIvyTest {
+
+  private PlayerRepo players;
+
+  @Test
+  void useRepo() {
+    var all = players.findAll().toList();
+    assertThat(all).isEmpty();
+
+    var james = new Player();
+    james.setFirstName("James");
+    james.setLastName("Bond");
+    james.setBirthDate(new Date(1968, 4, 13));
+
+    players.insert(james);
+    assertThat(players.findAll().toList())
+        .hasSize(1);
+    var persisted = players.findAll()
+        .filter(p -> "James".equals(p.getFirstName()))
+        .findFirst();
+    assertThat(persisted).isPresent();
+    assertThat(persisted.get().getId())
+        .isNotNull();
+
+    players.delete(persisted.get());
+    assertThat(players.findAll().toList())
+        .isEmpty();
+  }
+
+  @BeforeEach
+  void setUp() {
+    this.players = new PlayerRepo_(session("league"));
```

Comment:
jakarta.inject:2 is yet missing. I think later we should also plan a story to make injection of JPA items possible. 

something like this would be nice:
```
@Inject
@Named("league")
PlayerRepo players;
```